src/main/java/org/jenkinsci/test/acceptance/po/OicAuthSecurityRealm.java

```diff
@@ -15,9 +15,9 @@ public void configureClient(String clientId, String clientSecret) {
         control("clientSecret").set(clientSecret);
     }
 
-    public void setAutomaticConfiguration(String wellKnownEndpoint) {
-        control(by.radioButton("Automatic configuration")).click();
-        control("wellKnownOpenIDConfigurationUrl").set(wellKnownEndpoint);
+    public void setWellKnownEndpoint(String wellKnownEndpoint) {
+        control("").select("Discovery via well-known endpoint"); // Select doesn't have the relative path, directly path=/securityRealm/
```

Comment:
`control("")` is not right.  at this point you should introduce a new PageArea for the describable ("wellKnown)" and tag it with `@Describable`

https://github.com/jenkinsci/acceptance-test-harness/blob/master/docs/PAGE-OBJECTS.md#page-area

Reply:
done: https://github.com/jenkinsci/acceptance-test-harness/pull/1724/commits/1d8a510f771b9d82c111acb8a862738a4de9ba57

I didn't know I could nest `PageArea`s object that way